InfluxDb.cpp

```diff
@@ -76,24 +84,31 @@ boolean Influxdb::write(InfluxData data) { return write(data.toString()); }
  * for a list of error codes.
  */
 boolean Influxdb::write(String data) {
-  Serial.print(" --> writing to " + _db + ":\n");
-  Serial.println(data);
+  if (_debug == true) {
+    Serial.print(" --> writing to " + _db + ":\n");
+    Serial.println(data);
 
-  int httpResponseCode = http.POST(data);
-  Serial.print(" <-- Response: ");
-  Serial.print(httpResponseCode);
+    int httpResponseCode = http.POST(data);
```

Comment:
This part need to be outside of `if-debug` - otherwise the post will only be executed of debug is enabled.

Reply:
True, that is why I have included it into the `else` (starting at line 104) part of `if(_debug)` as well. It is pretty much duplicated without the Serial statements, I know that is not the most elegant solution but I could not decide on a better way.

Reply:
I think we should not duplicate the logic but only wrap logging in an if statement.